### PR TITLE
docs(template-syntax): post-RC5 Dart resync

### PR DESCRIPTION
**Pull request by chalin**
- Single line change to TS prose ([ignore whitespace diffs](https://github.com/angular/angular.io/pull/2121/files?w=1)).
- Dart-side code changes are to match a May 2016 TS-side commit: https://github.com/angular/angular.io/commit/ae2d8f27306c940bc127560649eb7fda8c10f291.
- Contributes to #2077.

Suites passed:
 - public/docs/_examples/template-syntax/dart
 - public/docs/_examples/template-syntax/ts

### CODE REVIEW

**Comment by chalin** [comment]
@kwalrath @Foxandxss : ready for review. TS side changes are trivial.

File: public/docs/ts/_cache/guide/template-syntax.jade [hunk 1]
```diff
@@ -858,8 +858,21 @@ block style-property-name-dart-diff
...
+      package in our Angular module. We add it to the `NgModule` decorator's `imports` array. This array contains the list
+      of external modules used by our application.
+      <br>Learn more about the `FormsModule` and `ngModel` in the
+      [Forms](../guide/forms.html#ngModel) chapter.
+
+  +makeExample('template-syntax/ts/app/app.module.1.ts', '', 'app.module.ts (FormsModule import)')
+
 :marked
-  Alternatively, we can use the canonical prefix form:
+  Alternatively to using `[(ngModel)]`, we can use the canonical prefix form:
```
Comment (wardbell) [inline public/docs/ts/_cache/guide/template-syntax.jade]: I think it is time to strike this alternative.

File: public/docs/ts/_cache/guide/template-syntax.jade [hunk 2]
```diff
@@ -858,8 +858,21 @@ block style-property-name-dart-diff
   header [()] = banana in a box
   :marked
     To remember that the parentheses go inside the brackets, visualize a *banana in a box*.
+
++ifDocsFor('ts|js')
+  .callout.is-important
+    header FormsModule is Required to use ngModel
+    :marked
+      Before we can use `ngModel` two-way data binding, we need to import the `FormsModule`
```
Comment (wardbell) [inline public/docs/ts/_cache/guide/template-syntax.jade]: Before we can use the `ngModel`  directive in a two-way data binding we must import the `FormsModule` .

File: public/docs/ts/latest/guide/template-syntax.jade [hunk 3]
```diff
@@ -859,16 +859,17 @@ block style-property-name-dart-diff
...
-    Before we can use `ngModel` two-way data binding, we need to import the `FormsModule`
-    package in our Angular module. We add it to the `NgModule` decorator's `imports` array. This array contains the list
-    of external modules used by our application.
-    <br>Learn more about the `FormsModule` and `ngModel` in the
-    [Forms](../guide/forms.html#ngModel) chapter.
++ifDocsFor('ts|js')
+  .callout.is-important
+    header FormsModule is Required to use ngModel
+    :marked
+      Before we can use `ngModel` two-way data binding, we need to import the `FormsModule`
```
Comment (wardbell) [inline public/docs/ts/latest/guide/template-syntax.jade]: See my comment above

File: public/docs/ts/latest/guide/template-syntax.jade [hunk 4]
```diff
@@ -859,16 +859,17 @@ block style-property-name-dart-diff
...
+      package in our Angular module. We add it to the `NgModule` decorator's `imports` array. This array contains the list
+      of external modules used by our application.
+      <br>Learn more about the `FormsModule` and `ngModel` in the
+      [Forms](../guide/forms.html#ngModel) chapter.
 
-+makeExample('template-syntax/ts/app/app.module.1.ts', '', 'app.module.ts (FormsModule import)')
+  +makeExample('template-syntax/ts/app/app.module.1.ts', '', 'app.module.ts (FormsModule import)')
 
 :marked
```
Comment (wardbell) [inline public/docs/ts/latest/guide/template-syntax.jade]: Consider striking

**Comment by wardbell** [comment]
I can make these small edits as I merge.

**Comment by chalin** [comment]
Yes please.

**Comment by chalin** [comment]
Just ensure that the _cache and latest copies of the ts .jade are the same.
I usually edit the latest and copy it over to the cache.

**Pull request closed without merging**